kinto_http/tests/test_session.py

```diff
@@ -14,6 +14,13 @@ def fake_response(status_code):
     response.status_code = status_code
     return response
 
+def find_headers():
+    kinto_http_version = pkg_resources.get_distribution("kinto_http").version
+    requests_version = pkg_resources.get_distribution("requests").version
+    python_version = '.'.join(map(str, sys.version_info[:3]))
+    header_info  = 'kinto-http.py/{} requests/{} python/{}'.format(kinto_http_version , requests_version, python_version)
+    return header_info
```

Comment:
I believe you can just import the values: `from kinto_http.session import USER_AGENT`